flask_monitoringdashboard/main.py

```diff
@@ -16,8 +16,9 @@
 
 dashboard.config.version = '3.2'
 dashboard.config.group_by = '2'
-dashboard.config.database_name = 'sqlite:///data.db'
-# dashboard.config.database_name = 'mysql+pymysql://user:password@localhost:3306/db1'
+dashboard.config.database_name = 'mysql://root:my-secret-pw@127.0.0.1:3306/fmd'
```

Comment:
is this a better connection string than the other ones?

Reply:
this is the module for quick testing of the FMD right? 
for testing probably it'd be simpler with sqlite: you don't have to create a DB in mysql...or?

Reply:
Argghh, I am using a mysql database for development and I keep accidentally pushing this file. I should make a copy of this file. It's definitely supposed to be the sqlite connection string! Good catch :)